src/AppBundle/Entity/Post.php

```diff
@@ -133,13 +133,13 @@ public function setAuthorEmail($authorEmail)
     /**
      * Is the given User the author of this Post?
      *
-     * @param User $user
+     * @param User|string $user
```

Comment:
We are not authenticated because that is a smoke test that guarantees that all private URLs redirect to the login form if you are unauthenticated.